develop-docs/development-infrastructure/devservices.mdx

```diff
@@ -84,23 +84,65 @@ devservices up --mode symbolicator
 
 ## Running a dependency locally
 
-If you want to run a dependency locally rather than as a container, you can do so by toggling the runtime to LOCAL.
+You can run a dependency locally instead of as a container by toggling its runtime via the `devservices toggle` command.
 
-For example, if you are running Sentry and want to use your local Snuba, you can do the following:
+### Toggle to local runtime
+
+For example, if you're running Sentry and want to run Snuba **locally** from source, you can toggle Snuba to the local runtime like this:
+
+```shell
+devservices toggle snuba
+```
+
+or you can specify the runtime explicitly:
 
 ```shell
 devservices toggle snuba LOCAL
 ```
 
-This will tell devservices to not bring up snuba and its dependencies, allowing you to run snuba locally instead.
+This tells devservices to treat Snuba as a service that should be started alongside dependent services like Sentry. Dependencies of Snuba, such as Redis, Kafka, and Clickhouse, will still run as containers.
+
+<Alert title="Note">
+  Toggling to LOCAL does not start the dev server for that service. You'll need to start that manually.
+</Alert>
+
+### What happens when you toggle
+
+If Sentry is already running, toggling Snuba to LOCAL will:
+
+  1. Stop Snuba's containers (unless shared by another service)
+  2. Treat Snuba as a local service
```

Comment:
I think we can leave this bullet point out, thoughts?

Reply:
fair